### PR TITLE
Update Contribution Map

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

* Remove 404 pages
* Update TiDB / TiKV / TiKV Client content
* Replace SIG info with contact channel

### CODE REVIEW

File: maps/contribution-map.md [hunk 1]
```diff
@@ -71,12 +71,12 @@ Contribution tutorial:
 
 ## TiKV Clients
 
-| *Module* | *Required Skills* | *Learning Materials* | *What I can Contribute* |
-| ---- | ---- | ---- | ---- |
-| [Rust Client](https://github.com/tikv/client-rust) | Rust, TiKV concepts, Transaction model | <li>[TiKV documentation](https://tikv.org/docs/3.0/concepts/overview/)<li>[Rust book](https://doc.rust-lang.org/book/)<li>[Practical networked applications in Rust](https://github.com/pingcap/talent-plan)<li>[Percolator paper](https://research.google/pubs/pub36726/) | [Issues](https://github.com/tikv/client-rust/issues) |
-| [Go Client](https://github.com/tikv/client-go) | Golang, TiKV concepts, Transaction model | Ditto | [Issues](https://github.com/tikv/client-go/issues) |
-| [Java Client](https://github.com/tikv/client-java) | Java, TiKV concepts, Transaction model | Ditto | |
-| [C Client](https://github.com/tikv/client-c) | C/C++, TiKV concepts, Transaction model | Ditto| [Issues](https://github.com/tikv/client-c/issues) |
```
Comment (sunxiaoguang) [inline maps/contribution-map.md:79]: Just want to double check is this removed intentionally?

Comment (tisonkun) [reply maps/contribution-map.md:79]: C client is actually used in TiFlash only and has many hack logics. The (to be) formal c/c++ client is https://github.com/tikv/client-cpp, but I think TiKV teams are currently working on rust, go, java and python clients only.

Comment (tisonkun) [reply maps/contribution-map.md:79]: cpp and node js clients are in a very early stage.

Comment (sunxiaoguang) [reply maps/contribution-map.md:79]: Understood.

**Comment by sunxiaoguang** [comment]
LGTM

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**